databases/TimescaleDb.java

```diff
@@ -1,14 +1,8 @@
-package databases;
-
-import datasets.Dataset;
-import query.QueryTranslator;
-import query.sql.SqlQueryTranslator;
-
 import java.sql.DriverManager;
 
-public class TimescaleDb implements Database{
+public class TimescaleDb implements Database {
     private String connUrl = "jdbc:postgresql://localhost:5432/bdspro?user=postgres&password=123qweasdzx";
-    private QueryTranslator queryTranslator = new SqlQueryTranslator();
```

Comment:
I think `QueryTranslator` should stay there. It will be easier to replace it later in the future if such need arises (but probably will not hah).

Or is there a reason for that?